PopPUNK/visualise.py

```diff
@@ -62,6 +62,9 @@ def get_options():
                              'from poppunk_assign [default = use that in the directory '
                              'of the query database]',
                         type = str)
+    iGroup.add_argument('--use-network',
```

Comment:
`--network-dir` would be a better name. And just pointing directly to the file likely to be better overall, this lets you pick different ranks, core/accessory only fits (in which case `--network-file`)

Reply:
Done in 1b40461.